src/webmon_app/reporting/dasmon/views.py

```diff
@@ -36,8 +36,12 @@ def dashboard(request):
     """
     # Get the system health status
     global_status_url = reverse(settings.LANDING_VIEW, args=[])
+    instrument_data = view_util.get_instrument_status_summary()
+    cutoff = int(len(instrument_data) / 2)
```

Comment:
In python 3, I think `//` is for integer division.

Reply:
yes, but I like the more explicit casting to `int`. You rarely see the `//` operator these days